### PR TITLE
added test for Falcon framework

### PR DESCRIPTION
**Pull request by jiri-one**
I have promised that long time ago, sorry that I am late ... :).

### CODE REVIEW

File: tests/fixtures/falcon_app/app.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a home page of Falcon app.</h3>
+            <div>
+            <a href='/second_page/'>LINK</a> to second page.
+            </div>
+        </body>
+    </html>"""
```
Comment (nappex) [inline tests/fixtures/falcon_app/app.py:20]: output files,which are generating by us should end with empty line. I supposed it is called EOF or something like that.
```suggestion
    </html>
"""
```

File: tests/fixtures/falcon_app/app.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        resp.text = """
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a second page of Falcon app.</h3>
+            <a href='/image_page/'>LINK to image page</a>
+        </body>
+    </html>"""
```
Comment (nappex) [inline tests/fixtures/falcon_app/app.py:33]:
```suggestion
    </html>
"""
```

File: tests/fixtures/falcon_app/app.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a image page of Falcon app.</h3>
+            <img src="/images/smile.png" alt="smile">
+            <a href="/">LINK to homepage</a>
+        </body>
+    </html>"""
```
Comment (nappex) [inline tests/fixtures/falcon_app/app.py:46]:
```suggestion
    </html>
"""
```

Comment (jiri-one) [reply tests/fixtures/falcon_app/app.py]: I know that David, but I was so lazy to fix that and I have hoped, that nobody will notice that :D. But you have one of your last free evenings and you are bored maybe, so I have to fix that .... :). Thanks for your comment ;).

**Comment by nappex** [comment]
😄 it shows me github. I was iterested how looks like falcon

File: tests/fixtures/falcon_app/app.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+# SOME NOTES:
+
+# you can run this Falcon app standalone with this part of code:
+# if __name__ == "__main__":
+#     from wsgiref.simple_server import make_server
+#     with make_server('', 8000, app) as httpd:
+#         print("Serving on port 8000...")
+#         httpd.serve_forever()
```
Comment (encukou) [inline tests/fixtures/falcon_app/app.py:99]: What would break if this wasn't commented?

Comment (jiri-one) [reply tests/fixtures/falcon_app/app.py:99]: I think absolutely nothing :). But since it's not elsewhere, I put it there only as a comment.

Comment (encukou) [reply tests/fixtures/falcon_app/app.py:99]: Flask sites run with `flask run`, so there's no code to put in.

Comment (jiri-one) [reply tests/fixtures/falcon_app/app.py:99]: I mean with "elsewhere" like in bottle_app. If you want, I can uncomment that, not a problem.

**Review by encukou — APPROVED** [review]

**Pull request merged**